README.md

```diff
@@ -40,7 +39,6 @@ iex> Req.new(url: "/fact", base_url: "https://catfact.nijna/")
 iex> import CurlReq
 ...> ~CURL(curl https://www.google.com)
 ...> # |> Req.request!()
-
```

Comment:
Do the doctests work without the newlines? I suppose I should add CI to the project.

Reply:
I added CI, so if you do a rebase then it'll run for this PR.

Reply:
I modified the CI to also run on PRs ;)

Reply:
Thanks, was wondering why it hadn't triggered :see_no_evil:

Reply:
You were right, the newlines are necessary 👍